app/js/openmrs.js

```diff
@@ -0,0 +1,119 @@
+angular.module('openmrs.resources', [ 'ngResource' ])
```

Comment:
Could you please follow [Style [Y010](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y010)]?